### ISSUE TITLE
Версии/Ошибки/Предложения/Вопросы-4

### ISSUE THREAD

**Issue by pumPCin** [pumPCin/GiveawayJoiner#32]

**Comment by pumPCin** [comment]
v1.5.4b-1.5.4c

1. Начиная с v1.5.4b изменено имя пользователя, в связи с этим переименуйте по пути ...\giveawayjoinerdata\Partitions\ имя папки "GiveawayJoiner" в "user" для сохранения всех настроек и данных из прошлой версии при обновлении.
2. Добавлен аргумент к запуску приложения "-userdata=имя_папки" с выбором рабочей папки, если нет аргумента то рабочая папка остается "giveawayjoinerdata" как и в прошлых вериях. Т.е. теперь можно не делать несколько копий всего приложения для разных учеток, а запускать сделав ярлык приложения и добавив первый аргумент к запуску определенной рабочей папки.
3. Сервис ИГ - добавлено сохранение данных о раздачах в которые вступили, чтобы сократить время на повторные попытки вступления в раздачу, когда сайт ИГ тупит и нет данных от него что уже вступили в раздачу.
4. Сервис IndieDB - добавлена опция  "Только Steam раздачи" .
5. Сервис ОП - добавлена опция  "Мин. количество участников".
6. Добавлена кнопка "Остановить ВСЕ сервисы" для одновременной остановки всех сервисов.
7. Сервис ЗП - исправления в Steam OAuth.

**Comment by solomad** [comment]
Максимально неудачное расположение кнопки "Остановить ВСЕ сервисы" =)

**Comment by pumPCin** [comment]
> Максимально неудачное расположение кнопки "Остановить ВСЕ сервисы" =)

Пока в тестовом режиме)  Добавил куда было легче всего это сделать, позже подумаю куда ее поместить...

**Comment by bolexator** [comment]
Что то странно работает через одну рабочую папку и ярлыки. Из четерых аккаунтов запустился один, остальные просто висят на поиске раздач.

**Comment by pumPCin** [comment]
> Что то странно работает через одну рабочую папку и ярлыки. Из четерых аккаунтов запустился один, остальные просто висят на поиске раздач.

Версию последнюю 1.5.4e запускаете? Если не входили в стим то во вкладке профиля вместо имени пользователя пишет имя рабочей папки аккаунта, т.е. для каждого аккаунта своя рабочая папка отображается?

**Comment by bolexator** [comment]
> > Что то странно работает через одну рабочую папку и ярлыки. Из четерых аккаунтов запустился один, остальные просто висят на поиске раздач.
> 
> Версию последнюю 1.5.4e запускаете? Если не входили в стим то во вкладке профиля вместо имени пользователя пишет имя рабочей папки аккаунта, т.е. для каждого аккаунта своя рабочая папка отображается?

Пробовал и 1.5.4e и 1.5.4d. Пользуюсь только IndieGala, в другие сервисы не заходил даже. Решилось странным образом: скопировал и заменил файлы из 100% рабочего аккаунта в папки где не работало. Заново сделал авторизацию и всё завелось.
Если всё делать начисто, то тот аккаунт где всё работало - снова работает нормально, а там где не работало - так же перестаёт работать. После замены файлов всё решается.

**Comment by bolexator** [comment]
Начиная с версии 1.5.4h, памяти стало больше потреблять. На процентов 20-25.

**Comment by Slejpnir** [comment]
Перестало входить в раздачи на Scrap.tf

**Comment by Deadmanovi4** [comment]
> Перестало входить в раздачи на Scrap.tf
Попробуй войти на scrap.tf в браузере. Скорее всего пермач за "бота" словил.

**Comment by Slejpnir** [comment]
Бана нет, вручную входит.

**Comment by AmazingFantastic** [comment]
Господа, как у вас с раздачами на Zeepond? Раньше я хоть что-то выигрывал, сейчас вообще голяк. В раздачи бот вступает постоянно, а выигрыша нет.
Админы сайта, увеличили время раздач что ли:? Кто столкнулся с таким/похожим:?

**Comment by bolexator** [comment]
Да и на индигала опять отвалилось. Вроде работало нормально на новой версии, потом стал показывать уровень -1, теперь не входит в раздачу, пишет - Пропуск - невозможно вступить из-за условий раздачи (banned). Вручную в раздачи в вступает.

**Comment by bolexator** [comment]
Уважаемый разработчик! Просим починить сервис Indiegala. Что то опять они накрутили.

**Comment by pumPCin** [comment]
> Уважаемый разработчик! Просим починить сервис Indiegala. Что то опять они накрутили.

Рад бы, только меня давно забанили акки на индигала, до этого на скрапе, потом еще на стимгифте, поэтому и версий больше не выходило с тех пор

**Comment by bolexator** [comment]
> > Уважаемый разработчик! Просим починить сервис Indiegala. Что то опять они накрутили.
> 
> Рад бы, только меня давно забанили акки на индигала, до этого на скрапе, потом еще на стимгифте, поэтому и версий больше не выходило с тех пор

Давайте поможем как-нибудь. Я лично, пользуюсь только индигала и стимгифтс.

**Comment by pumPCin** [comment]
> > > Уважаемый разработчик! Просим починить сервис Indiegala. Что то опять они накрутили.
> > 
> > 
> > Рад бы, только меня давно забанили акки на индигала, до этого на скрапе, потом еще на стимгифте, поэтому и версий больше не выходило с тех пор
> 
> Давайте поможем как-нибудь. Я лично, пользуюсь только индигала и стимгифтс. Могу для индигалы аккаунтов дать пару штук. Там у них есть бан на одновременное использование в раздачах больше одного аккаунта (только в раздачах) - пришлось впн попользовать. Бан могут дать на неделю или на месяц. Перманентный ещё не получил ни разу.

можно попробовать, сам я давно создавал новые, они жили буквально день, потом их отрубали, просто ошибка писалась без указания бана, далее сейчас попробовал зайти в старые акки и уже пишет что выдан бан, если будет не нужный акк, могу попробовать попровить индигалу, надеюсь что не по ip заблочили

**Comment by bolexator** [comment]
Мои рабочие ip (статичные) забанены, с них нельзя выходить. Если создать новый и зайти в раздачу, будет сразу бан. Дома работает, поскольку меняются адреса периодически. Но всё равно, стал пользоваться впном теперь постоянно.

**Comment by Extroom** [comment]
> можно попробовать, сам я давно создавал новые, они жили буквально день, потом их отрубали, просто ошибка писалась без указания бана, далее сейчас попробовал зайти в старые акки и уже пишет что выдан бан, если будет не нужный акк, могу попробовать попровить индигалу, надеюсь что не по ip заблочили

Если IP заблокировано, то сразу бан прилетит и безразлично это новый, или старый акк. Мой домашний статический IP в бане и бывало несколько случаев, когда VPN глючил и выключился, то сразу прилетал бан, но на сутки, потом через VPN работало. Правда, на двух акках таки заблокировали, но только площадку с раздачами и теперь там не дает вступить, а все остальное работает.
Поэтому можно попробовать зайти на старые акки через VPN, возможно какой-нибудь будет работать, если нет, то создать новый. Главное на каждый акк заходить с разного IP и user agent.

**Comment by bolexator** [comment]
Заработала. Спасибо!

**Comment by bolexator** [comment]
Нет, всё такие есть проблема: ищет раздачи только на первых двух страницах. 
![e816a220b3](https://user-images.githubusercontent.com/82199583/219674452-d0ae9e8a-d0d2-494a-8fff-90b7465d7df6.jpg)

**Comment by Extroom** [comment]
тоже аналогичная проблема

**Comment by bolexator** [comment]
Уважаемый разработчик. Снова просим вас исправить баг. Раздачи ищутся только на 2 страницах, вне зависимости от настроек.

**Comment by bolexator** [comment]
И кстати ещё баг - после прохождения раздач, не чекает на выигрышные ключи. Приходится самому заходить и проверять.

**Comment by pattername** [comment]
Просьба к разработчику: расскажите об ситуации с программой. Будет доработка или остается как есть? Потому что действительно, индигала не функционирует.

**Comment by bolexator** [comment]
Товарищ разработчик, будут исправления? Посдкажите.

**Comment by CrazyGhostRider** [comment]
Здравствуйте
tf2r.com закрылся.
Админ предлагает использовать scrap.tf и [mannco.store](https://mannco.store/giveaways)
